src/compile/data/stack.ts

```diff
@@ -191,6 +192,10 @@ export class StackNode extends DataFlowNode {
     }, {});
   }
 
+  public hash() {
+    return 'StackNode' + hash(this._stack);
```

Comment:
`\`Stack ${hash(this._stack)}\``

Reply:
colon?

Reply:
I don't care but @kanitw suggested that we already use colons in the hash. It doesn't really matter so I'd say space is fine.

Reply:
It looks nicer in the test without colon lol -- not that it's a big deal.